packages/buildbucket-dart/pubspec.yaml

```diff
@@ -7,9 +7,10 @@ environment:
   sdk: '>=3.0.0-0 <4.0.0'
 
 dependencies:
-  protoc_plugin: 21.1.2
-  fixnum: 1.1.0
-  protobuf: 3.1.0
+  protoc_plugin: ^21.1.2
+  fixnum: ^1.1.0
+  protobuf: ^3.1.0
+  grpc: ^3.2.4
```

Comment:
Could we pin the specific versions and leave the version bump to the dependabot?

Reply:
Yeah I can do that. I was trying to get rid of warnings but it appears it does not matter.